etc/calibre.profile

```diff
@@ -22,7 +22,7 @@ nonewprivs
 noroot
 nosound
 novideo
-protocol unix,inet,inet6
+protocol unix,inet,inet6,netlink
```

Comment:
Hi - let's leave netlink disabled for now. It's an easy fix to change this from a user's perspective, and I think it's better to err on the side of too much restriction here.